adding documentation for query null semantics (#2865)
dotnet/EntityFramework.Docs: entity-framework/core/querying/user-defined-function-mapping.md

FROM [Posts] AS [p]
```

> ## Configuring nullability of user-defined function based on it's arguments

Reviewer: ```suggestion
## Configuring nullability of user-defined function based on its arguments
```

> If the user-defined function can only return `null` when one or more of it's arguments are `null` EFCore provides way to specify that, which results in more performant SQL. It can be done by adding a `PropagatesNullability()` call to the relevant function parameters model configuration.

Reviewer: ```suggestion
If the user-defined function can only return `null` when one or more of its arguments are `null`, EFCore provides way to specify that, resulting in more performant SQL. It can be done by adding a `PropagatesNullability()` call to the relevant function parameters model configuration.
```
